ui/components/prssection/pressection_test.go

```diff
@@ -0,0 +1,99 @@
+package prssection
+
+import (
+	"testing"
+
+	"github.com/dlvhdr/gh-dash/data"
+)
+
+func Test_excludeArchivedPullRequests(t *testing.T) {
+	m := &Model{}
+
+	tests := []struct {
+		name     string
+		prs      []data.PullRequestData
+		expected int
+	}{
+		{
+			name: "Not all pull requests are archived",
+			prs: []data.PullRequestData{
+				{
+					Repository: struct {
+						NameWithOwner string
+						IsArchived    bool
+					}{
```

Comment:
Can you extract the struct type to a type variable?

Reply:
Thanks for review! I fixed it, is that ok?

https://github.com/dlvhdr/gh-dash/pull/199/commits/d34c2aac16fe51c3e633a908d14cc1f46f2eb523